deployment/scripts/clusters.backup-multiple-servers.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# time stamp
+printf "\nMultiple backups started at:\n  "
+date
+
+# base setup
+SCRIPT_PATH=$(realpath $0)
+SCRIPT_DIR=$(dirname $SCRIPT_PATH)
+
+# save old CONFIGURATION for later reset
+export SAVE_CONFIGURATION=$CONFIGURATION
+
+# export all variables in "../.env"
+set -a            
+source ${SCRIPT_DIR}/../.env
+set +a
+
+# check BACKUP_CONFIGURATIONS
+if [[ -z ${BACKUP_CONFIGURATIONS} ]]; then
+  #%! echo "You must provide a BACKUP_CONFIGURATIONS via environment variable"
+  printf "!!! You must provide a BACKUP_CONFIGURATIONS via environment variable !!!\n"
+  exit 1
+else
+  # convert configurations to array
+  IFS=' ' read -a CONFIGURATIONS_ARRAY <<< "$BACKUP_CONFIGURATIONS"
+
+  # display the clusters
+  printf "Backup the clusters:\n"
+  for i in "${CONFIGURATIONS_ARRAY[@]}"
+  do
+    echo "  $i"
+  done
+fi
```

Comment:
You can remove the else block in most cases since you exit in the if block. The mechnic should be:

```
if(error_condition1){
  exit
}
continue execution command1
continue execution command2
continue execution command3
...
if(error_condition2){
  exit
}
...
```

Reply:
Done for both cases